test/verify/test_arg_ops.cpp

```diff
@@ -2,34 +2,54 @@
 #include "verify_program.hpp"
 #include <migraphx/program.hpp>
 #include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
 #include <migraphx/op/argmax.hpp>
 #include <migraphx/op/argmin.hpp>
 
-template <class T, int Axis>
-struct test_arg_ops : verify_program<test_arg_ops<T, Axis>>
+template <class T, int Axis, bool Transpose>
+struct test_arg_ops : verify_program<test_arg_ops<T, Axis, Transpose>>
```

Comment:
For the operators that are changed, I would suggested added a case with broadcasted input shape test case.
Maybe another case of sliced input shape.